pandas/tests/io/parser/test_common.py

```diff
@@ -83,6 +86,7 @@ def test_empty_decimal_marker(all_parsers):
         parser.read_csv(StringIO(data), decimal="")
 
 
+@skip_pyarrow
```

Comment:
What percentage of tests in this module are we skipping? It might make more sense to reorganize into classes and apply this skip to the class rather than marking a ton of tests individually

Reply:
It's a lot (15% of the total) so it may be worth doing. Will look

Reply:
@arw2019 if possible can you xfail instead of skip, that way if the tests somehow pass in the future, we can find and un xfail them.

Reply:
its totally fine for now to xfail these, just as long as we have a consistent name)